stl/inc/format

```diff
@@ -536,15 +536,23 @@ protected:
     }
 
     _Fmt_codec_base() {
+#pragma warning(push)
+#pragma warning(disable : 4640) // '_Cvt_cache': construction of local static object is not thread-safe

```

Comment:
This warning indicates that you're using Magic Statics. (It's confusingly phrased - what's happening is that the STL's test suite disables Magic Statics and enables this warning with `/w14640 /Zc:threadSafeInit-`. This ensures that anything that *would* be thread-safe via Magic Statics, instead emits this warning.)

We avoid Magic Statics in the STL: https://github.com/microsoft/STL/issues/673#issuecomment-608170434

Thus I believe we need another mechanism here.